src/scripts/farming/Farming.ts

```diff
@@ -120,7 +120,7 @@ class Farming implements Feature {
 
         if (!suppressNotification) {
             Notifier.notify({
-                message: `You earned ${money} money from the harvest!`,
+                message: `You earned ${money} pokédollars from the harvest!`,
```

Comment:
It looks like our standard is to have this capitalized like you did in the other notification

Reply:
Fixed it